README.md

```diff
@@ -35,6 +35,8 @@ Read [this blog post on functional ASP.NET Core](https://dusted.codes/functional
     - [routeCif](#routecif)
     - [routeStartsWith](#routestartswith)
     - [routeStartsWithCi](#routestartswithci)
+    - [subRoute](#subRoute)
+    - [subRouteCi](#subRouteCi)
```

Comment:
This should be #subroute and #subrouteci, because GitHub generates lowercase IDs and they are case sensitive.